composer.json

```diff
@@ -137,8 +139,9 @@
     "codeception/codeception": "^2.4.0",
     "codeception/phpunit-wrapper": "^7.0",
     "phpstan/phpstan": "^0.7",
-    "symplify/easy-coding-standard-tester": "^4.4",
-    "symplify/changelog-linker": "^4.8"
+    "symplify/easy-coding-standard-tester": "^5.2.7",
+    "symplify/changelog-linker": "^5.2.7",
+    "symplify/monorepo-builder": "^5.2.7"
```

Comment:
Bump for `symplify/*` package is needed, so new `ReleaseWorkerInterface` feature of `symplify/monorepo-builder` can be used for release process.

They mutually depend on each other, so every other `symplify/*` package of the same version will be installed.